### PR TITLE
Update external links in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by futhr**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ in order to craft an excellent pull request:
    git push origin <topic-branch-name>
    ```
 
-7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
+7. [Open a Pull Request](https://help.github.com/articles/about-git-rebase/)
```
Comment (chrismccord) [inline CONTRIBUTING.md]: This link should stay as is, no?

Comment (futhr) [inline CONTRIBUTING.md]: It's an 301 redirect from an old link, but same content.

Comment (chrismccord) [inline CONTRIBUTING.md]: https://help.github.com/articles/about-pull-requests/ is not a 301 and is a different article than https://help.github.com/articles/about-git-rebase/ . What am I missing? :)

Comment (futhr) [inline CONTRIBUTING.md]: True indeed.

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Pull request merged**